### PR TITLE
Made the necessary changes minus one

### PR DESCRIPTION
**Pull request by puttagunta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Welcome to our technical test! 
 
-First, if you are here it means that you are moving forward, this is good! Now you have the chance to show us how good you are to be a part one of the best software test team. 
+First, if you are here it means that you are moving forward, this is good! Now you have the chance to show us how good you are to be a part of the best software test team. 
 
 Before you start take a big breath and… 
 - Relax
```
Comment (RussellLMiller) [inline README.md]: Good idea

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 Welcome to our technical test! 
 
-First, if you are here it means that you are moving forward, this is good! Now you have the chance to show us how good you are to be a part one of the best software test team. 
+First, if you are here it means that you are moving forward, this is good! Now you have the chance to show us how good you are to be a part of the best software test team.
```
Comment (aayyathan) [inline README.md]: Now you have a chance to show us how good you are!

File: README.md [hunk 3]
```diff
@@ -19,8 +19,8 @@ About the recruitment process:
 - Job offer
 
 # Commit rules
-- Create a branch for your work. To make things simple, create branches following the naming convention yourname-date-branchname
-- Create a pull request and let the interviewers know the name of the branch
+- Create a branch for your work. To make things simple, create branches following the naming convention yourname-date
+- Create a pull request and choose _aayyathan_ as the reviewer
```
Comment (aayyathan) [inline README.md]: select

File: README.md [hunk 4]
```diff
@@ -19,8 +19,8 @@ About the recruitment process:
...
 - Feel free to add another readme, if you need to provide any explanation on the exercise
```
Comment (aayyathan) [inline README.md]: , if you need let us know any additional information/comments on this test.

**Pull request merged**